packages/shared/friends/sagas.ts

```diff
@@ -198,14 +200,24 @@ function* configureMatrixClient(action: SetMatrixClient) {
   }
 
   // initialize conversations
-  client.onStatusChange((socialId, status) => {
+  client.onStatusChange(async (socialId, status) => {
```

Comment:
who catches errors in this function?

Reply:
Refactored the function so that it will never throw an error